src/Sylius/Bundle/UiBundle/Tests/Functional/TemplateEventTest.php

```diff
@@ -34,7 +34,7 @@ public function it_renders_template_events_blocks(): void
     {
         // See Kernel.php for the configuration resulting in those lines
         $expectedLines = [
-            'First block (for event: "first_event")',
```

Comment:
Removed `_event` variable - it wasn't used anywhere but in the tests and after introducing multiple events it would be quite misleading.